apps/studio/components/interfaces/Integrations/Wrappers/CreateIcebergWrapperSheet.tsx

```diff
@@ -292,18 +318,21 @@ export const CreateIcebergWrapperSheet = ({
                         </div>
                       )}
                       <div className="flex flex-col gap-2">
-                        <Label_Shadcn_ className="text-foreground-light">
-                          Target Schema
-                        </Label_Shadcn_>
-                        <SchemaSelector
-                          portal={false}
-                          size="small"
-                          selectedSchemaName={values.target_schema}
-                          onSelectSchema={(schema) => setFieldValue('target_schema', schema)}
-                          onSelectCreateSchema={() => setCreateSchemaSheetOpen(true)}
+                        <InputField
```

Comment:
for iceberg specifically - does this mean each wrapper needs to be in its own schema?